cuckoo/web/controllers/analysis/control/api.py

```diff
@@ -35,6 +36,7 @@
 
 class ControlApi(object):
     @staticmethod
+    @csrf_exempt
```

Comment:
I think keeping the `@csrf_exempt` allow an attacker to perform a CSRF attack. The protection was added for security purpose. In addition, you made it possible sending CSRF tokens from front-end part (JS). Thus, in my mind, `@csrf_exempt` isn't needed here.